### ISSUE TITLE
Update or remove gruntPatternlab()

### ISSUE THREAD

**Issue by mortona42** [forumone/generator-web-starter-gesso#22]
```javascript
    gruntPatternlab : function() {
      if (typeof this.options.getPlugin === "function" && this.options.getPlugin('grunt')) {
        // Add copy task for Pattern Lab
        var copy = this.options.getPlugin('grunt').getGruntTask('copy');
        copy.insertConfig('copy.patternlabStyleguide', this.fs.read(this.templatePath('tasks/patternlab/copy.js')));
        copy.loadNpmTasks('grunt-contrib-copy');
        this.options.addDevDependency('grunt-contrib-copy', '^0.8.0');

        // Watch task for Pattern Lab
        var watch = this.options.getPlugin('grunt').getGruntTask('watch');
        watch.insertConfig('watch.patternlab', this.fs.read(this.templatePath('tasks/patternlab/watch.js')));
        watch.loadNpmTasks('grunt-contrib-watch');
        watch.loadNpmTasks('grunt-simple-watch');
        this.options.addDevDependency('grunt-contrib-watch', '^0.6.1');
        this.options.addDevDependency('grunt-simple-watch', '^0.1.2');

        // Shell task for Pattern Lab
        var shell = this.options.getPlugin('grunt').getGruntTask('shell');
        shell.insertConfig('shell.patternlab', this.fs.read(this.templatePath('tasks/patternlab/shell.js')));
        shell.loadNpmTasks('grunt-shell');
        this.options.addDevDependency('grunt-shell', '^1.1.2');

        this.options.getPlugin('grunt').registerTask('buildPatternlab', [{
          task : 'copy:patternlabStyleguide',
          priority : 1
        },
        {
          task : 'shell:patternlab',
          priority : 2
        }]);

        //build => build.js buildPatternlab==task name 100 priority
        this.options.getPlugin('grunt').registerTask('build', 'buildPatternlab', 100);
      }
    },
```

**Comment by wwhurley** [comment]
The removal of the questions for SASS and Pattern Lab should be made to master and then to a 0.2 release since it would be a breaking change for 0.1-based projects.

**Comment by wwhurley** [comment]
Resolved as part of 0.2
